styles.css

```diff
@@ -1047,3 +1047,96 @@ ul {
   margin-top: 26px;
   margin-right: 36px;
 }
+
+#contact {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  background-image: url(images/contact.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: top right;
+  background-color: #6070ff;
+  margin-top: 40px;
+  border-top-left-radius: 80px;
+}
+
+#contact h2 {
+  color: #fff;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  margin-bottom: 0;
+}
+
+#contact p {
+  color: #ebebff;
+  font-size: 20px;
+  line-height: 28px;
+  margin-left: 24px;
+  margin-right: 24px;
+  vertical-align: top;
+}
+
+#form {
+  display: flex;
+  justify-content: center;
+}
+
+#form ul {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.form-control {
+  width: 327px;
+  height: 48px;
+  border-radius: 8px;
+  background-color: #fff;
+  color: #cfd8dc;
+  margin: 0 auto;
+  font-size: 17px;
+  font-weight: 200px;
+  border: none;
+  padding-left: 10px;
+}
+
+.inner-list li:nth-child(2) {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.form-textarea {
+  height: 170px;
+  font-size: 15px;
+  color: #172b4b;
+}
+
+.submit-btn {
+  text-align: left;
+}
+
+.btn-submit {
+  color: #6070ff;
+  background-color: #fff;
+  border: none;
+  font-size: 17px;
+  font-weight: 500px;
+  line-height: 24px;
+  letter-spacing: 3%;
+  border-radius: 8px;
+  padding: 12px 16px;
+  margin-top: 15px;
+  margin-bottom: 30px;
+}
+
+.btn-submit:hover {
+  transform: scale(0.98);
+  box-shadow: 3px 2px 22px 1px rgba(0, 0, 0, 0.24);
+}
```

Comment:
- [ ] Kindly, make sure that Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template. check the screenshot below:
![image](https://user-images.githubusercontent.com/71874425/186142037-41a9e629-218f-4ab5-9f54-cf90ae740945.png)